Library/Homebrew/cmd/bottle.rb

```diff
@@ -300,69 +301,78 @@ def bottle_formula(f)
     puts "./#{filename}"
     puts output
 
-    if ARGV.include? "--rb"
-      File.open("#{filename.prefix}.bottle.rb", "w") do |file|
-        file.write("\# #{f.full_name}\n")
-        file.write(output)
+    if ARGV.include? "--json"
+      json = {
+        f.full_name => {
+          "formula" => {
+            "pkg_version" => f.pkg_version.to_s,
+            "path" => f.path.to_s.gsub("#{HOMEBREW_REPOSITORY}/", ""),
```

Comment:
@UniqMartin This way will include the tap path so I don't think it _needs_ to be either. You'll note we don't include the tap as a separate field; it's actually not needed for anything beyond the path.